object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,10 @@
+/* exported toObject */
+function toObject(keyValuePair) {
+  var object = { test: 'test' };
+  for (var x in object) {
+    x = keyValuePair[0];
+    object[x] = keyValuePair[1];
+  }
+  delete object.test;
+  return object;
+}
```

Comment:
Why do we need to have `object` with `test: 'test'` and then delete it?? Just wondering. If it's just for testing purpose, let's remove those